tests/rule_based_profiler/bobster_user_workflow_verbose_profiler_config.yml

```diff
@@ -0,0 +1,36 @@
+# This profiler is meant to be used on the NYC taxi data:
+# tests/test_sets/taxi_yellow_trip_data_samples/yellow_trip_data_sample_20(18|19|20)-*.csv
+variables:
+  # BatchRequest yielding thirty five (35) batches (January, 2018 -- November, 2020 trip data)
+  jan_2018_thru_nov_2020_monthly_trip_data_batch_request:
+    datasource_name: taxi_pandas
+    data_connector_name: monthly
+    data_asset_name: my_reports
```

Comment:
I think it's an antipattern to have batch details baked into these configs. The profiler rules should be easily/frequently resusable.

Reply:
@jcampbell I am in favor of this, too.  We just need to figure out exactly how the "golden triumvirate" will be passed to the `Profiler`, without it being present in the YAML configuration.  In addition, we must provide for being able to override any and all member of the "golden triumvirate" via the configuration, because it is conceivable for some `ParameterBuilder` configurations to require this flexibility (e.g., training set from a different data source).  Thanks.

Reply:
We came up with an approach at Architecture Review on 6/24/2021 whereby we will pass the "golden triumvirate" to the `Profiler` constructor.  If this is not urgent, how about we move this task into the scope of the very next Epic?